msal/acquire_token_silent_parameters.go

```diff
@@ -9,34 +9,33 @@ import (
 )
 
 // AcquireTokenSilentParameters contains the parameters to acquire a token silently (from cache).
-type acquireTokenSilentParameters struct {
-	commonParameters *acquireTokenCommonParameters
+type AcquireTokenSilentParameters struct {
+	commonParameters acquireTokenCommonParameters
 	account          AccountProvider
 	requestType      requests.RefreshTokenReqType
-	clientCredential *msalbase.ClientCredential
+	clientCredential msalbase.ClientCredential
 }
 
 // CreateAcquireTokenSilentParameters creates an AcquireTokenSilentParameters instance with an empty account.
 // This can be used in the case where tokens are acquired as the application instelf.
-func createAcquireTokenSilentParameters(scopes []string) *acquireTokenSilentParameters {
-	p := &acquireTokenSilentParameters{
+func CreateAcquireTokenSilentParameters(scopes []string) AcquireTokenSilentParameters {
+	p := AcquireTokenSilentParameters{
```

Comment:
nit: couple other places simplified these style of returns, could do the same here

Reply:
done